src/app/Fake.BuildServer.TeamCity/TeamCity.fs

```diff
@@ -235,6 +373,40 @@ module TeamCity =
         /// The Build number assigned to the build by TeamCity using the build number format or None if it's not on TeamCity.
         static member BuildNumber = Environment.environVarOrNone "BUILD_NUMBER"
 
+        /// Get the branch of the main VCS root
+        static member Branch with get() = BuildParameters.tryGetConfiguration "vcsroot.branch"
```

Comment:
If we add these to the environment then we might add the other APIs from `BuildParameters` as well?

Reply:
Possible but I thing it's nice that this class/module only contains specific parameters, consumers here don't care that it came via the java parameters file or via environment variables as long as it's something team city make available